index.html

```diff
@@ -18,18 +31,18 @@
     <![endif]-->
 
   </head>
-  <body>
+  <body >
 
     <div class="container">
       <div class="pycon-logo text-center">
-        <img src="pycon_india_2017.png" style="max-width: 75%;">
+        <img src="pycon_india_2017.png" style="max-width: 75%;"></img>
```

Comment:
ok. I will keep that in mind but in webmatrix the another tag comes automatically . so sorry .